contributing/code/patches.rst

```diff
@@ -90,20 +99,25 @@ Check that the current Tests Pass
 Now that Symfony is installed, check that all unit tests pass for your
 environment as explained in the dedicated :doc:`document <tests>`.
 
-Step 2: Work on your Patch
---------------------------
+.. tip::
+
+    If tests are failing, check on `Travis-CI`_ if the same test is
+    failing there as well. In that case you do not need to be concerned
+    about the test failing locally.
+
+Step 3: Work on your PR
+-----------------------
 
 The License
 ~~~~~~~~~~~
 
-Before you start, you must know that all the patches you are going to submit
-must be released under the *MIT license*, unless explicitly specified in your
-commits.
+Before you start, you must know that all the code you are going to submit
+must be released under the *MIT license*.
```

Comment:
not sure, but I think the double "must" makes it sound a bit off, maybe use will instead?

```suggestion
will be released under the *MIT license*.
```

Reply:
@dbrumann I reworded the first `must` instead